corehq/apps/app_manager/models.py

```diff
@@ -5947,7 +5950,86 @@ def save(self, *args, **kwargs):
         super(GlobalAppConfig, self).save(*args, **kwargs)
 
 
+class LatestEnabledAppReleases(models.Model):
+    domain = models.CharField(max_length=255, null=False)
+    app_id = models.CharField(max_length=255, null=False)
+    location = models.ForeignKey(SQLLocation, on_delete=models.CASCADE, to_field='location_id')
+    build_id = models.CharField(max_length=255, null=False)
+    version = models.IntegerField(null=False)
+    active = models.BooleanField(default=True)
+    activated_on = models.DateTimeField(null=True)
+    deactivated_on = models.DateTimeField(null=True)
+
+    @staticmethod
+    def expire_cache(domain, location_id, app_id):
```

Comment:
Are you implying that this is better as a util method than being in the model?
That would give me motivation to move this to util and not because its being used in an util method.

Reply:
I don't know why this is part of the model at all. In my mind, the purpose of a staticmethod is to group functions related to a particular class but that don't need an instance (self) or the class (cls). This method acts on a function method that is far away from this class (even though that util method uses the class). 
I would either:
- move that util into this class as a staticmethod
- move this method to be next to the function that it acts on

Reply:
Did the following,
[moved logic to util](https://github.com/dimagi/commcare-hq/pull/23680/commits/6ef2bebb701a67161c1af9997a500556b0bf5dff)
[used signal to address cache expire](https://github.com/dimagi/commcare-hq/pull/23680/commits/6ef2bebb701a67161c1af9997a500556b0bf5dff)